CMakeLists.txt

```diff
@@ -34,6 +34,8 @@ get_filename_component(VVL_INCLUDE_DIR ${VVL_INCLUDE_DIR} DIRECTORY)
 # This reflects the current structure of the VulkanSDK.
 target_include_directories(Vulkan::Headers INTERFACE ${VVL_INCLUDE_DIR})
 
+find_package(VulkanLayerUtils REQUIRED CONFIG)
```

Comment:
`VulkanLayerUtils` doesn't have `find_package` support yet.

Reply:
Ah, I wasn't undertanding why it wasn't working! \o/

Do you have a reference for how it's done in another repository?